pkgs/development/libraries/quarto/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, lib
+, pandoc
+, esbuild
+, deno
+, fetchurl
+, makeWrapper
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "quarto";
+  version = "1.0.38";
+
+  src = fetchurl {
+    url = "https://github.com/quarto-dev/quarto-cli/releases/download/v${finalAttrs.version}/quarto-${finalAttrs.version}-linux-amd64.tar.gz";
+    sha256 = "sha256-bqEGRX6ihyrgmIxqkV9jr/2QfWM5LWnsjoGpTqK9qOc=";
+  };
+
+  buildInputs = [
+      pandoc
+      deno
+      esbuild
+      makeWrapper
+  ];
+
+  patches = [
+      ./fix-deno-path.patch
+  ];
+
+  preFixup = ''
+      wrapProgram $out/bin/quarto --prefix PATH : ${lib.makeBinPath [ deno ]};
+  '';
+
+  installPhase = ''
+      runHook preInstall
+
+      mkdir -p $out/bin
+      mkdir -p $out/share
+
+      rm bin/tools/deno
+      rm bin/tools/pandoc
+      rm bin/tools/esbuild
```

Comment:
Remove `bin/tools` at once?

Reply:
No, because there are mandatory sub-directories with executables.Only this three executables ca be used through nixpkgs.